src/upnp.py

```diff
@@ -236,22 +237,20 @@ def run(self):
                 self.deletePortMapping(router)
         shared.extPort = None
         if deleted:
-            shared.UISignalQueue.put(('updateStatusBar', tr._translate("MainWindow",'UPnP port mapping removed')))
+            shared.UISignalQueue.put(('updateStatusBar', tr._translate("MainWindow", 'UPnP port mapping removed')))
         logger.debug("UPnP thread done")
 
     def getLocalIP(self):
         s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         s.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+        # FIXME: Contacting Google's DNS as a privacy oriented tool isn't a wise idea.
+        # Maybe use some shell script for that job.
         s.connect((uPnPThread.GOOGLE_DNS, 1))
         return s.getsockname()[0]
 
     def sendSearchRouter(self):
         from debug import logger
-        ssdpRequest = "M-SEARCH * HTTP/1.1\r\n" + \
-                    "HOST: %s:%d\r\n" % (uPnPThread.SSDP_ADDR, uPnPThread.SSDP_PORT) + \
-                    "MAN: \"ssdp:discover\"\r\n" + \
-                    "MX: %d\r\n" % (uPnPThread.SSDP_MX, ) + \
-                    "ST: %s\r\n" % (uPnPThread.SSDP_ST, ) + "\r\n"
+        ssdpRequest = "M-SEARCH * HTTP/1.1\r\n" + "HOST: %s:%d\r\n" % (uPnPThread.SSDP_ADDR, uPnPThread.SSDP_PORT) + "MAN: \"ssdp:discover\"\r\n" + "MX: %d\r\n" % (uPnPThread.SSDP_MX, ) + "ST: %s\r\n" % (uPnPThread.SSDP_ST, ) + "\r\n"
```

Comment:
Thank you @mirrorwish for helping with the review.

Perhaps this could be reformatted entirely, by doing something like:

"""protocol line 1
protocol line 2
protocol line 3""" % (variable1, variable2, variable3)

?